cql3/statements/describe_statement.cc

```diff
@@ -716,6 +745,18 @@ future<std::vector<std::vector<bytes_opt>>> schema_describe_statement::describe(
                 schema_result.insert(schema_result.end(), ks_result.begin(), ks_result.end());
             }
 
+            if (_with_internals) {
+                const auto auth_service_ptr = client_state.get_auth_service();
+                // TODO: Fail in a more delicate way here, i.e. return a proper exception.
+                SCYLLA_ASSERT(auth_service_ptr);
```

Comment:
everywhere we do *client_state.get_auth_service() I think we can stick to this simplification